### PR TITLE
Article accessibility

### LINKED ISSUE #84: ADR: Accesibility

### PR DESCRIPTION
**Pull request by johnnolan**
Fixes #84

### CODE REVIEW

File: src/research-development/articles/accessibility-practices.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+> The Web Content Accessibility Guidelines are an internationally recognised set of recommendations for improving web accessibility.
+
+Any services created will be built to the WCAG 2.1 standard.
+
+We will also ensure as newer versions of the standard get released that we meet these as part of our ongoing commitment to continuous improvement.
+
+We will do this by
+
+* Ensure all professions collaborate on early designs
+* [Adding automation to our pipelines](#automated-testing-in-pipelines)
```
Comment (andrewpearce-digital) [inline src/research-development/articles/accessibility-practices.md:37]:
```suggestion
* [Adding automated testing to our pipelines](#automated-testing-in-pipelines)
```

File: src/research-development/articles/accessibility-practices.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+* Adding checks in our development environments
```
Comment (andrewpearce-digital) [inline src/research-development/articles/accessibility-practices.md:38]: What's the difference between adding checks and automated testing?

Comment (johnnolan) [reply src/research-development/articles/accessibility-practices.md:38]: More to do with built in tooling like vscode extensions and potential pre-commit hooks. Will change the wording but also tempted to just remove it, what are your thoughts, too much information and keep with automation?

Comment (johnnolan) [reply src/research-development/articles/accessibility-practices.md]: I have removed this after consideration, I think its too much and is assumed from other areas of the document

**Review by hawx — APPROVED** [review]

**Pull request merged**